### ISSUE TITLE
Magento 2.4 with PHP 7.4

### ISSUE THREAD

**Issue by valenciaisaza** [bazaarvoice/magento2-extension#69]
Error during the Setup:di:compile process
 
```
Deprecated Functionality: Unparenthesized `a ? b : c ? d : e` is deprecated. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /Applications/MAMP/htdocs/lightsonline/vendor/bazaarvoice/bazaarvoice-magento2-ext/ViewMo  
  del/Pixel.php on line 192
```

**Comment by gabriel3iri** [comment]
will this be fixed any time soon? having the same issue here

**Comment by svalveson** [comment]
We are having the same issue here. There is an official fix?

**Comment by valenciaisaza** [comment]
The error was fixed on this PR https://github.com/bazaarvoice/magento2-extension/commit/0a98343634f377a48dff40d1211296cb0725b841

**Comment by jantzenw** [comment]
Hi,

Thank you for opening this issue. Sorry for the delay. This issue should be fixed in v8.2.1. Please upgrade to that version and let us know if the issue is not resolved.
